Makefile

```diff
@@ -102,3 +102,12 @@ container-push:
 .PHONY: docs
 docs:
 	$(MAKE) -C $(REPO_ROOT)/documentation build
+
+.PHONY: test-e2e
+test-e2e:
+	@echo "Beginning end-to-end tests";
+	@echo "Checking if the API is running...";
+	@curl --silent --output /dev/null localhost:8000 || (echo "Failed to connect to the API. Is it running?"; exit 1;)
+	@echo "Running end-to-end tests";
+	@HI_CONFIG=file://$(REPO_ROOT)/config/test_config.json go test $(BASE_PACKAGE)/services/user/tests/end_to_end || exit 1;
```

Comment:
Since the api is started before the `HI_CONFIG` environment variable is set to test config here, the api would still be using the dev config